steps/discover-repos.rb

```diff
@@ -51,6 +51,11 @@
 raise 'Can only retrieve up to 1000 repos' if opts[:total] > max
 
 size = [opts[:page_size], opts[:total]].min
+licenses_to_filter = [
```

Comment:
@Orillio we should provide this list from the command line. Now they are hardcoded, which makes it impossible to configure.

Reply:
@yegor256 I wanted to propose this as the issue for other students. Since there are not many (easily solvable) issues in this repo right now. 
As you said, the course isn't about solving tasks, but about open source collaboration/discussions.